### PR TITLE
[DOCS] UI changes: Alerts and Exceptions pages

### LINKED ISSUE #1396: [DOCS] UI changes: Alerts and Exceptions pages
## Description 

Update docs for Alerts page and Exceptions page for a handful of UI changes observed in 8.0.0. Also look for related issues or PRs to make sure we cover any new functionality related to these changes.

### Alerts page
- Added "Data view" drop-down to top right toolbar (may end up cropping this out, since it's not strictly speaking part of the Alerts page)
- Refresh button is lighter blue now
- Count and Trend histogram drop-downs changed from `signal.rule.name` to `kibana.alert.rule.name`
- Other fields also changed from "signal.rule.x" to "kibana.rule.x" or "kibana.alert.rule.x", but they're not immediately visible in UI (no change in table header labels) -- check if they're referenced in procedures, etc.

### Exceptions page
- Removed "Number of rules assigned to" column
- Minor UI tweaks like spacing between columns
 
## Sample Screenshots
Images taken from 8.0.0-rc1.

### Alerts page
image

### Exceptions page
image

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #1396.

Previews:
- [Elastic Security UI](https://security-docs_1397.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html)
- [Detections and alerts](https://security-docs_1397.docs-preview.app.elstc.co/guide/en/security/master/detection-engine-overview.html)
- [Manage detection alerts](https://security-docs_1397.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html) (2nd image from top of page)
- [Rule exceptions and value lists | View and manage exception lists](https://security-docs_1397.docs-preview.app.elstc.co/guide/en/security/master/detections-ui-exceptions.html#manage-exceptions)

### CODE REVIEW

**Comment by marshallmain** [comment]
I looked through the preview pages and had a few questions - not sure if these are in scope for this PR or not, but I just scanned through each entire preview page:

### Detections and Alerts page
  - Should we switch to calling them "alerts" in the docs for 8.0? cc @jethr0null 
image
### Manage Detection Alerts page
  - `.siem-signals-<space name>` is now `.alerts-security.alerts-default` in 8.0. This is a breaking change, however, we added index aliases so that the `.siem-signals-<space name>` will also work on the new alerts indices for the time being. I can work with someone on the docs team to document the breaking changes and mitigations if that's not done already. In addition, in the 3rd screenshot the fields being searched on are now `kibana.alert.building_block_type` and `kibana.alert.rule.type: 'threat_match'`
image
image
image
### Rule exceptions and value lists page**
  - `signal.original_event.*` -> `kibana.alert.original_event.*`
image

File: docs/detections/detections-ui-exceptions.asciidoc [hunk 1]
```diff
@@ -274,7 +274,7 @@ The Exceptions table enables you to view and manage all exceptions that have bee
 [role="screenshot"]
 image::images/exceptions-page.png[]
 
-Exceptions are automatically grouped into exception lists. The table displays each exception list on an individual row, with the most recently created list at the top. Each row contains information such as the number and name of rule(s) the exception list is assigned to, the name of the rule(s) assigned to the exception list, the dates the list was created and last edited, and options to export or delete it.
+Exceptions are automatically grouped into exception lists. The table displays each exception list on an individual row, with the most recently created list at the top. Each row contains information such as the name of rule the exception list is assigned to, the dates the list was created and last edited, and options to export or delete it.
```
Comment (benironside) [inline docs/detections/detections-ui-exceptions.asciidoc:277]:
```suggestion
Exceptions are automatically grouped into exception lists. The table displays each exception list on an individual row, with the most recently created list at the top. Each row contains information such as the name of the rule to which the exception list is assigned, the dates when the list was created and last edited, and options to export or delete it.
```
```suggestion
Exceptions are automatically grouped into exception lists. The table displays each exception list on an individual row, with the most recently created list at the top. Each row contains information such as the name of rule the exception list is assigned to, the dates the list was created and last edited, and options to export or delete it.
```

**Review by benironside — COMMENTED** [review]
One minor suggestion, otherwise LGTM. Especially liked the screenshot where you removed the menus on the left 👏

**Review by nastasha-solomon — APPROVED** [review]

**Review by marshallmain — APPROVED** [review]
Looks great, thanks!

File: docs/detections/detection-engine-intro.asciidoc [hunk 2]
```diff
@@ -73,7 +73,7 @@ Cold tier is a {ref}/data-tiers.html[data tier] that holds time series data that
 
 {es-sec} does *NOT* support cold tier data for the following {es} indices:
 
-* Index patterns controlled by {es-sec}, including signals and list indices
+* Index patterns controlled by {es-sec}, including alerts and list indices
```
Comment (jmikell821) [inline docs/detections/detection-engine-intro.asciidoc:76]: We've had some rendering issues with `es-sec` in the past, so this is just a small nit. 
```suggestion
* Index patterns controlled by {elastic-sec}, including alerts and list indices
```

**Review by jmikell821 — COMMENTED** [review]
One super small attribute nit, then when Marshall OKs this I think it's good to go!

**Pull request merged**